packages/xod-client-electron/arduino-libraries/LiquidCrystal_I2C/examples/BlinkingCursor/BlinkingCursor.ino

```diff
@@ -0,0 +1,32 @@
+#include <Wire.h> 
+#include <LiquidCrystal_I2C.h>
```

Comment:
Why include these examples? 🤔

Reply:
It’s a clone of the library from GitHub as is. Technically we don’t need them. But should we really modify 3-rd party vendored sources? I think it would complicate maintenance (i.e. library updates).